### PR TITLE
Organization Culture

### PR DESCRIPTION
**Pull request by steve-fenton-octopus**
# About

This is a post about the history of modern workplace culture and how this is important to DevOps.

# Release

This is planned to go out after the page https://octopus.com/devops/culture/ is live. I'll keep it as a draft until then.

# Blog image idea

I've used an images from Spaces, so we might want to choose a different one?

# Pre-requisites

- [x] The draft is complete and this post is ready to be reviewed.
- [x] I'm familiar with the [writing for Octopus TL;DR](https://www.octopus.design/writing/writing-tldr/).
- [x] The PR build passes validation, and if it doesn't, I've checked the [common validations errors](https://style.octopus.com/writing-at-octopus-tldr#common-validation-errors) and none of those apply.
- [x] The images I've included follow the [rules for working with images](https://www.octopus.design/writing/working-with-images/).
- [x] I've added alt text for all the images I've included (alt text describes image to people unable to see it, 125 characters max)
- [x] One screenshot (if any are included) is suitable for Twitter. This screenshot should be 16:9 ratio and at least 1200 x 675 pixels. For screenshots from the Octopus Web Portal, please use Chrome and the browser’s zoom function to remove as much white space as possible. If screenshots are annotated (e.g., with boxes or arrows), please also provide a version that is unannotated for use on Twitter. Do not use dark mode for the Octopus Web Portal.

## How to review this PR

Just the usual :)

## Publication date

TBC

### CODE REVIEW

File: blog/2023-01/key-assumption-work-culture/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+You'll find culture is the toughest nut to crack in DevOps. It's tempting to rely on research and statistics to prove the case for a generative culture. Still, the reality is that cultural change depends on compelling storytelling and creating a compelling vision of what the organization will look like after the transition.
+
+You can read more about DevOps culture [in our DevOps engineer's handbook](https://octopus.com/devops/)
+
+## Further reading
+
+Related information:
+
+- [Culture in The DevOps engineer's handbook](https://octopus.com/devops/culture/)
```
Comment (steve-fenton-octopus) [inline blog/2023-01/key-assumption-work-culture/index.md:128]:
```suggestion
- [The DevOps engineer's handbook](https://octopus.com/devops/)
```

Comment (steve-fenton-octopus) [reply blog/2023-01/key-assumption-work-culture/index.md:128]: @TeganAli I don't think the culture content will arrive in time, so this makes the blog safe to go live. We can circle back and deep link to culture content later.

**Comment by TeganAli** [comment]
Ok great, thanks so much for that @steve-fenton-octopus :)

**Review by TeganAli — APPROVED** [review]

File: blog/2023-01/key-assumption-work-culture/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+- Climate for learning
+- Westrum organizational culture
+- Psychological safety
+- Job satisfaction
+- Identity
+
+The cultural capabilities drive software delivery and operations performance, which predict successful business outcomes.
+
+![The headline DevOps capabilities, showing culture as a driver for software delivery performance](simplified-structural-equation-model-2022.png)
```
Comment (TeganAli) [inline blog/2023-01/key-assumption-work-culture/index.md:27]:
```suggestion
![The headline DevOps capabilities showing culture as a driver for software delivery performance](simplified-structural-equation-model-2022.png)
```

**Review by TeganAli — APPROVED** [review]

**Pull request merged**